### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by Seva55**

### CODE REVIEW

File: src/redux/actions.ts [hunk 1]
```diff
@@ -10,6 +11,7 @@ const actions = {
   EDIT_THING,
   COMPLETE_THING,
   DELETE_THING,
+  SESSION_THING,
```
Comment (regrunge) [inline src/redux/actions.ts]: Maybe call this ADD_SESSION? or it is a bit confusing 😆

File: src/redux/reducers/things.ts [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
+
+      clone.splice(selectedThingIndex, 1, action.thing);
+
+      return clone;
+    case actions.DELETE_THING:
+      const selectedThingIndexTBD = clone.findIndex((t) => t.id === action.id);
+      clone.splice(selectedThingIndexTBD, 1);
+
+      return clone;
+    case actions.SESSION_THING:
```
Comment (regrunge) [inline src/redux/reducers/things.ts]: create its own file in src/redux/reducers/sessions.ts with all actions for the session

File: src/redux/actions.ts (hunk 1 above)
Comment (Seva55) [reply src/redux/actions.ts]: Ok, I thought they should be the same Thing

File: src/redux/reducers/things.ts [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
-            // DO STUFF
-
-            return clone;
-        default:
-            return state;
-    }
+  const clone = [...state];
+
+  switch (action.type) {
+    case actions.ADD_THING:
```
Comment (regrunge) [inline src/redux/reducers/things.ts:8]: copy this action to do ADD_SESSION ;)

**Pull request merged**